Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -89,7 +128,7 @@ def render_GET(self, request):
             neighbor_level = int(request.args["neighbor_level"][0])
 
         # TODO: Remove dummy return and change to aggregated data calculation
```

Comment:
Please move this dummy data to an external class. That way, this method can be tested fully.